src/LikeButton.jsx

```diff
@@ -108,7 +108,7 @@ const likeClick = () => {
 const title = hasLike ? "Unlike" : "Like";
 
 return (
-  <LikeButton disabled={state.loading || dataLoading || !context.accountId} title={title} onClick={likeClick}>
```

Comment:
is this still necessary?

Reply:
I think so, otherwise a like can be added by a user that isn't logged in.